modules/ROOT/pages/6.4.1-release-notes.adoc

```diff
@@ -0,0 +1,1138 @@
+
+=== Inserting elements that was not valid within the closest editing host would incorrectly split the editing host
+//#TINY-9595
+
+In previous versions of {productname}, inserting invalid child blocks into a parent block element would create issues with the editing host element.
+
+As a consequence, the editing host element would incorrectly split itself into multiple elements. 
+
+To fix this issue, with the release of {productname} 6.4.1, pasted elements are now instead unwrapped inside the target element rather than being wrapped.
+
+In {productname} 6.4.1, pasting blocks that are invalid children of a `contenteditable="true"` blocks will no longer split that block, but instead unwrap the blocks to fit within the specific `contenteditable="true"` block.
+
+=== color_cols option was not respected in the forecolor or backcolor color swatches
+//#TINY-9560
+
+In previous versions of {productname}, the logic for specifying the number of columns in the `forecolor` and `backcolor` swatches was updated with the release of {productname} 6.3.
+
+As a consequence, this logic didn't use the color_cols option in {productname} v6.3
+
+[source, js]
+----
+tinymce.init({
+    selector: "textarea",
+    plugins: "table code",
+    toolbar: "forecolor backcolor tableprops",
+    color_cols: 7, // <-- option was not respected
+});
+----
+
+In {productname} 6.4.1, the color swatches opened by the `forecolor` and `backcolor` toolbar buttons have the amount of columns set by the `color_cols` option, if it is set.
+
+=== Drag and dropping the last noneditable element out of its parent block would not properly pad the parent block element
+//#TINY-9606
+
+In previous versions of {productname}, dragging the last child element with a `contentedtiable="false"` attribute out of a block element would prevent the user from placing the caret back into that source block.
+
+As a fix, {productname} 6.4.1, added a new condition which validates if the source element is empty when the drop operation occurs. If it is empty then it will be padded with a `BR` element so that it's possible to place the caret inside the block. 
+
+In {productname} 6.4.1, dragging and dropping elements that are `contenteditable="false"` out of they're parent blocks now correctly pad the parent block element.
+
+=== Nonbreaking space do not get highlighted when enabling show invisible characters
+//#TINY-9685
+
+In previous versions of {productname}, it was discovered that nonbreaking spaces were not highlighting correctly when the "show invisible characters" plugin was enabled.
+
+This issue caused not only confusion with the intended text layout, but impacted the overall user experience. To resolve this issue, {productname} introduced a fix to ensure that all invisible characters, including nonbreaking spaces, are accurately displayed when the plugin is enabled.
```

Comment:
This is a confusing one to explain. Might need some rewording, there's too many negatives.

```suggestion
=== Non breaking spaces were not highlighted within a nested editable section
//#TINY-9685

In previous versions of {productname}, when using the "show invisible characters" feature of the `visualchars` plugin, non breaking spaces were not highlighting within an editable section that was nested inside a noneditable section.

To resolve this issue, {productname} 6.4.1 introduced a fix to ensure that all invisible characters, including nonbreaking spaces, are accurately displayed when the plugin is enabled.
```

Reply:
@TheSpyder Thanks for the above suggestion, It might be worth nothing that we are also trying to avoid using the term `noneditable` and `editable`, and replace with `contenteditable="false" or `contenteditable="true" from now on in the docs to be more consistent.